app/PropositionFactory.php

```diff
@@ -50,17 +50,16 @@ public function search($term, $pagination) {
 		
 		return $query;
 	}
-
-	public function getQueuedCount() {
-		return Proposition::whereStatus(2)->count();
-	}
 	
 	public function getQueuedPropositions() {
 		return Proposition::whereStatus(2)->orderBy('deadline', 'asc')->get();
 	}
 	public function getQueuedPropositionsExeptUsers($id) {
 		return Proposition::whereStatus(2)->whereNotIn('proposer_id', [$id])->orderBy('deadline', 'asc')->get();
 	}
+	public function getQueuedExceptUserCount($id) {
```

Comment:
Just to have clear functions, can you rename `getQueuedExceptUserCount` to `getQueuedPropositionsExceptUserCount`?